src/main/java/com/fasterxml/uuid/Jug.java

```diff
@@ -102,6 +102,10 @@ private static void printMap(Map<String,String> m, PrintStream out, boolean opti
 
     public static void main(String[] args)
     {
+        new Jug().run(args);
+    }
+
+    public void run(String[] args) {
```

Comment:
This is effectively the core of this change on this file, I destaticised most of the methods, the only one that is still static is main.

Take note that I have tried my best to not touch too much of the main file, just shuffling the methods around so it is easier to test.

Also take note that Intellij appears to have applied its own automated styling guidelines, so I would like to apologise in advance for the changes.